install/check-memcached-backend.sh

```diff
@@ -0,0 +1,17 @@
+echo "${_group}Checking memcached backend ..."
+
+if grep -q "\.PyMemcacheCache" "$SENTRY_CONFIG_PY"; then
+  echo "found PyMemcacheCache in $SENTRY_CONFIG_PY, looks good"
+else
+  if grep -q "\.MemcachedCache" "$SENTRY_CONFIG_PY"; then
```

Comment:
Wonder if in this case, we can't rename their existing `CACHES` variable to `CACHES_BAK` and print a message that we did so? This keeps from overwriting their settings wholesale, which might be quite different from what our recommended defaults were, and are not otherwise backed up. In particular, I'm worried about how any changes the user might have made to `OPTIONS` will interact with our change, since the Django Memcached and PyMemCached have very different options APIs.

Reply:
That sounds great, but imo we're out of time. We need to get a security release out the door and not ship the memcached breakage with it. :(

Reply:
@azaslavsky Do you think it would be better to simply hard-fail if they have `.MemcachedCache` in their config file, with some docs on how to manually address?

Reply:
Honestly, hard fail seems like the best way to go. If people are up in arms about it, we can always cut another release with one of these automatic config changes we're discussing. But using regex to try and guess at user's non-version-controlled configs is always risky imo.

I can write up the doc if you want to just change this PR to point the user to https://develop.sentry.dev/self-hosted/releases/#breaking-changes?

Reply:
Deal.

Reply:
Done in b8e8baa353bd2d8f4f2b0474154b9dc43adc4a14.